app/models/katello/concerns/operatingsystem_extensions.rb

```diff
@@ -21,7 +21,7 @@ def assign_templates!
               provisioning_template_name = Setting["katello_default_#{kind.name}"]
             end
 
-            if (template = ProvisioningTemplate.find_by_name(provisioning_template_name))
+            if (template = ProvisioningTemplate.unscoped.find_by_name(provisioning_template_name))
```

Comment:
After this change, it does appear to associate the templates, which is good. :)

Do we need also do the unscoped query for any other foreman resources used by this flow?  (e.g. Ptable, ...)